examples/devicefarm/testgridproject.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: devicefarm.aws.upbound.io/v1beta1
+kind: TestGridProject
+metadata:
+  annotations:
+    upjet.upbound.io/manual-intervention: "Requires manually add securityGroupIds and subnetIds"
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    name: example
+    region: us-west-2
+    vpcConfig:
+      - securityGroupIds: #${aws_security_group.example.*.id} - should be manually added all GroupId
+          - <securityGroupId1>
```

Comment:
Please add reference configuration and use cross-resource referencing here. See this as an example: https://github.com/upbound/provider-aws/blob/b560d85b36930aa23010f67583786b8c7fa78dfa/config/appstream/config.go#L22

Reply:
Done